postfix/recipes/service.rb

```diff
@@ -0,0 +1,4 @@
+service 'postfix' do
```

Comment:
Refactored this from `postfix::relay` mostly to follow a pattern we have elsewhere: a distinct `foo::service` recipe which defines the resource and is included elsewhere.